main.py

```diff
@@ -231,12 +226,13 @@ def mousePressed(self, event):
                     self.clickedButton = None
                 elif (self.clickedButton is not None and
                       self.clickedButton != SEND_WAVE_BUTTON):
-                    canBuyTower = \
-                        self.checkCanBuyTower(self.clickedButton.iconColor)
-                    if canBuyTower:
-                        self.towerButtonClicked = True
-                    else:
-                        self.towerButtonClicked = False
+                    self.towerButtonClicked = True
+                    # canBuyTower = \
```

Comment:
I think you should just remove these comments

Reply:
Done for debugging and forgot about it lol